Alerts/readme.md

```diff
@@ -87,7 +87,7 @@ $showAlert = trim(getContent(
 	"show:__customshowalertmessage__true",
 	"noecho"
 ));
-if ($showAlert == true){
+if ($showAlert == "true"){
```

Comment:
You can also just do a simple `if ($showAlert) {` here since an empty string will _return false_.